utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameter.kt

```diff
@@ -0,0 +1,10 @@
+package org.oppia.android.util.platformparameter
+
+/**
+ * Generic interface that is used to provide Platform Parameter Values corresponding to the
+ * [PlatformParameter] proto values. Objects that implement this interface will override the [value]
+ * property to store the actual platform parameter value.
+ * */
```

Comment:
```suggestion
 */
```

Reply:
This is neccessary for KDoc. So cannot remove this asterisk

Reply:
Issue created https://github.com/oppia/oppia-android/issues/3394

Reply:
Done